content/200-concepts/100-components/02-prisma-client/000-working-with-prismaclient/130-logging.mdx

```diff
@@ -65,6 +65,10 @@ To use event-based logging:
 
 The following example subscribes to all `query` events and write the `duration` and `query` to console:
 
+<TabbedContent tabs={[<FileWithIcon text="Relational databases" icon="database"/>, <FileWithIcon text="MongoDB" icon="database"/>]}>
+
+<tab>
+
 <CodeWithResult expanded="{true}">
 <cmd>
 
```

Comment:
In the line below this one, the end of the second highlight should be 26 and not 25.
<img width="411" alt="image" src="https://user-images.githubusercontent.com/183673/157941209-0899335a-efa6-4cd7-bd8a-b914095dde4d.png">

Reply:
Done